### DISCUSSION TITLE
Upscale or inpaint - Connection errored out

### DISCUSSION THREAD

**Discussion by tmprabubiz** [lllyasviel/Fooocus#475] [Q&A] [answered]
Whenever I try to upscale or do outpainting after pulling the desired image in to the input box and hit generate, the error message appears on the screen "Connection errored out"
In the command box the number "1006" appears

Please guide

**Comment by dennis0906** [comment]
Did you find out the problem? I have the same issue

**Comment by junevm** [comment, marked as answer]
same here

**Comment by xhoxye** [comment]
Please upload a full screenshot of your console
